kafka-connect/kafka-connect-runtime/main/LICENSE

```diff
@@ -1449,14 +1449,8 @@ License (from POM): Apache License, Version 2.0 - https://aws.amazon.com/apache2
 
 --------------------------------------------------------------------------------
 
-Group: org.mongodb Name: bson Version: 4.11
+Group: org.mongodb Name: bson Version: 4.11.5
 Project URL (from POM): https://github.com/mongodb/mongo-java-driver
 License (from POM): The Apache License, Version 2.0 - http://www.apache.org/licenses/LICENSE-2.0.txt
 
 --------------------------------------------------------------------------------
-
-Group: software.amazon.s3.analyticsaccelerator  Name: analyticsaccelerator-s3  Version: 1.0.0
```

Comment:
is this not used anymore? because it was just added

Reply:
It doesn't appear in the `lib` folder of the kafka-connect-runtime distributions.